firmware/raspi/gui.py

```diff
@@ -0,0 +1,353 @@
+"""Module containing main GUI code to interact with game.py.
+
+Based on https://github.com/saavedra29/chess_tk
+"""
+import random
+import sys
+import tkinter as tk
+from tkinter import simpledialog
+
+import chessboard
+from game import Game
+import player
+from util import parse_args
+
+class GUI:
+    """GUI class that creates a visual interface of the board.
+    """
+    pieces = {}
+    selected_piece = None
+    focused = None
+    turn = ""
+    move_first = ""
```

Comment:
Can you please change this to `human_plays_white_pieces` and make it hold `True` or `False` rather than a string?